src/hooks/useNotificationTypes.tsx

```diff
@@ -790,6 +792,60 @@ export const notificationTypes: NotificationTypeConfig[] = [
       };
     },
   },
+  {
+    type: NotificationType.PredictionMarketConcluded,
+    useTrigger: ({ trigger }) => {
+      const allFills = useAppSelector(getSubaccountFills, shallowEqual) ?? EMPTY_ARR;
+      const stringGetter = useStringGetter();
+
+      // TRUMPWIN-USD
+      const trumpWinFills = allFills.filter((fill) => fill.marketId === PREDICTION_MARKET.TRUMPWIN);
+      const trumpWinDeleveraged = trumpWinFills.filter(
+        (fill) => fill.type === AbacusOrderType.Deleveraged
+      );
+      const trumpWinOffset = trumpWinFills.filter(
+        (fill) => fill.type === AbacusOrderType.Offsetting
+      );
+
+      useEffect(() => {
+        if (trumpWinDeleveraged.length > 0 || trumpWinOffset.length > 0) {
+          const latestTrumpFill = [...trumpWinDeleveraged, ...trumpWinOffset].sort(
+            (a, b) => b.createdAtMilliseconds - a.createdAtMilliseconds
+          )[0];
+
+          const hadCorrectOutcome = latestTrumpFill?.type === AbacusOrderType.Offsetting;
```

Comment:
is offsetting guaranteed to happen when the market winds down?

Reply:
Yes. There will be either offsetting or deleveraged orderTypes present when a market enters final_settlement.